mypy/util.py

```diff
@@ -20,6 +21,15 @@
 ENCODING_RE = \
     re.compile(br'([ \t\v]*#.*(\r\n?|\n))??[ \t\v]*#.*coding[:=][ \t]*([-\w.]+)')  # type: Final
 
+PLAIN_ANSI_DIM = '\x1b[2m'  # type: Final
```

Comment:
Is this portable? Could we get this from `curses` or something?

Reply:
I tried it in couple terminals and it works (because it is ANSI standard). The problem is that although it is a basic ANSI "feature", terminfo files for most default terminals don't have `dim` termcap entry, so `curses` doesn't report it. I was thinking about choosing a grey color that would look good on both white and black background, but it is actually not easy, and again most default terminals are 8-color, not 256-color, so we can't get the color code from `curses`.

Reply:
Ok, sounds reasonable. Maybe add a comment about this?

Since this is not on by default, we can iterate on this afterwards even if some things don't work in every possible environment.